template/package.json.ejs

```diff
@@ -9,7 +9,6 @@
   },
   "keywords": [],
   "author": "",
-  "license": "BSD-3-Clause",
```

Comment:
perhaps we should also mark these as private. what do you think the defaults of the package.json for a new template should be @chrisdickinson ? this project is of course open source but these plug-ins my be private and commercial.

Reply:
We're probably looking for
```
  "license": "UNLICENSED",
  "private": "true",
```

("private" will keep `npm` and friends from ever publishing it!)

Reply:
cool, thanks !

Reply:
changing now